universe/api/api.go

```diff
@@ -2,7 +2,7 @@ package api
 
 import (
 	"context"
-
+	"github.com/gin-gonic/gin"
```

Comment:
let's have dependencies to be simple to look for, here is example
https://github.com/momentum-xyz/ubercontroller/blob/develop/database/worlds/db.go#L3-L13

Reply:
changed